aries-site/src/pages/templates/banner-notifications.mdx

```diff
@@ -1,3 +1,155 @@
+import { Anchor } from 'grommet';
+import {  BestPracticeGroup, Example } from '../../layouts';
+import {
+  BannerContentLayoutExample,
+  BannerNotificationCritical,
+  BannerNotificationCriticalClose,
+  BannerNotificationDoExample,
+  BannerNotificationDontExample,
+  BannerNotificationInfo,
+  BannerNotificationWarning,
+  BannerNotificationWarningClose,
+  BannerNotificationDiagram,
+} from '../../examples';
 
-<ComingSoon />
+
+<Example
+  showResponsiveControls={['fullScreen']}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  caption="An example of a dashboard with a Global Banner warning the user that their subscription is expiring. The user can either dismiss the notification or click on the anchor to renew their subscription."
+  screenContainer={{ scale: 0.57 }}
+>
+  <BannerContentLayoutExample />
+</Example>
+
+
+## Guidelines
+
+Global Banners are high-attention notifications used primarily for status updates on background processes and upcoming system changes and releases.
+
+Global Banners are used to showcase Warning and Critical statuses. They are also a mechanism for informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissable by the user.
+
+## Anatomy
+<Example
+  caption="Diagram illustrating the four parts composing of a Banner Notification."
+  plain
+>
+  <BannerNotificationDiagram />
+</Example>
+
+1. Status Indicator is an icon made of a shape and color to capture the user’s attention
+2. Message is a short description explaining the issue
+3. Anchors (optional)  are used to direct users to links related to the notification
+4. Close Button (optional) is used to dismiss the notification
+
+## Best Practices 
+### Placement
+- Always located directly below the Global Header without a gap and should not obscure any underlying content or action
+- Can persist across multiple sessions until dismissed if necessary
+
+### Format
+- Only show one banner at a time
+- Global Banners should stay within the 192 character limit
+- Long text is not truncated but instead will wrap on smaller screen sizes
+- Anchors are left-aligned and will wrap alongisde the text
+- Use only two or fewer anchors
+- Utilize writing guide to stay within character count as well as the writing effective content guidance for appropriate language
+
+### Behavior
+- Never sticky and should scroll when there is other content
+- The background spans the full width of the screen, but the content resizes alongside the Global Header
+
+
+## When should you use Global Banners?
+Since Global Banners take over the top of an interface and can be intrusive, they should be reserved only for important system generated updates. Also when applicable, Global Banners should always have a relevant anchor(s) to direct users to further actions or to view more about the issue in full detail.
```

Comment:
It seems this section should just be under "Guidelines"? and this header is unnecessary.